pkg/driver/nodeServer.go

```diff
@@ -926,6 +910,7 @@ func (s *NodeServer) mountVolume(devName, targetPath, fsType string, mountOption
         l.Errorf("Failed to mount device %+v. Error: %v", devName, err)
         return status.Error(codes.Internal, err.Error())
     }
+
```

Comment:
minor: extra line, can be removed.

Reply:
Although it is optional, I think it is a good visual separation of the "if" block from the rest of the code